MAINTAINERS.md

```diff
@@ -54,15 +54,18 @@ does not in itself necessitate a major release.
 To cut a new release:
 1. Determine if this is a [minor or major release](#versioning-scheme) and
    change the `version` field in `Cargo.toml` accordingly.
-2. Build the project to update `Cargo.lock`, then run `nix/update-carnix.sh` to
+1. Build the project to update `Cargo.lock`, then run `nix/update-carnix.sh` to
```

Comment:
Not sure about this. Yes, GitHub renders this as 1. 2. 3., but IMO Markdown is meant to be readable without any rendering at all. So I'm in favour of numbering lists 1. 2. 3. instead of 1. 1. 1.

Reply:
Sure, reverting.